src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container shadow p-3 mb-5 bg-light rounded">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <img src="assets/Ceph_Logo.svg"
+           alt="Ceph"
+           class="img-fluid mx-auto d-block">
+      <h3 class="text-center"
+          i18n>Welcome to Ceph</h3>
+
+      <div class="m-5">
+        <h4 class="text-center"
+            i18n>Please proceed to complete your Cluster Creation</h4>
```

Comment:
"the" 300k results, "your" 22k and [capitalization](https://www.patternfly.org/v4/ux-writing/capitalization):
```suggestion
            i18n>Please proceed to complete the cluster creation</h4>
```

Reply:
Done.